cmd/create.go

```diff
@@ -350,27 +337,31 @@ func (cc *createCmd) createDeployment() error {
 			deployType = cc.deployType
 		}
 
-		deployConfig, err := d.GetConfig(deployType)
+		deployConfig, err = d.GetConfig(deployType)
 		if err != nil {
 			return err
 		}
-		customInputs, err = prompts.RunPromptsFromConfigWithSkips(deployConfig, maps.Keys(flagVariablesMap))
+		for _, variable := range deployConfig.Variables {
+			fmt.Printf("Name %s, Value %s\n", variable.Name, variable.Value)
```

Comment:
is this print supposed to be a debug statement? it's unlikely to be necessary to print every value on each run